Practice/case.py

```diff
@@ -0,0 +1,86 @@
+"""case"""
+import re
+
+
+def get_input():
+    """
```

Comment:
Look at the example docstrings in [PEP-257](https://www.python.org/dev/peps/pep-0257/) and please use that format. There is no blank line before the docstring.